GETTING_STARTED.md

```diff
@@ -33,6 +33,20 @@ import { MaterialModule } from '@angular/material';
 export class PizzaPartyAppModule { }
 ```
 
+## Configuring SystemJS
```

Comment:
Can you move this section below ``### Additional setup for `md-slide-toggle` and `md-slider`:``?